packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2190,6 +2249,20 @@ export class ContainerRuntime
 	}
 
 	public setConnectionState(connected: boolean, clientId?: string) {
+		if (connected && this.idCompressorMode === "delayed" && !this.delayedCompressorLoading) {
```

Comment:
nit: Rename `this.delayedCompressorLoading` -> `this.delayedCompressorLoaded`. Currently, it's not clear from the name that it is used to know whether compressor has been loaded or not.

Reply:
Oh I see. It's called `Loading` because we started the load async. Maybe something like `this.compressorLoadinitiated`?